custom_components/opnsense/switch.py

```
@@ -10,241 +11,292 @@
-    # @property
-    # def is_on(self):
-    #     return False
+    @property
+    def available(self) -> bool:
+        return self._available
 
-    # @property
-    # def extra_state_attributes(self):
-    #     return None
+    # Move this to OPNsenseEntity once all entity-types are updated
+    async def async_added_to_hass(self) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)